src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -267,32 +268,32 @@ void primitive_inst::update_impl() {
     GPU_DEBUG_PROFILED_STAGE(instrumentation::pipeline_stage::update_implementation);
     auto prev_impl_str =  _impl != nullptr ? _impl->get_kernel_name() : "nullptr";
     if (!_node->is_type<data>() && !(_node->is_type<mutable_data>() && _node->get_dependencies().empty())) {
+        auto fake_params = _node->type()->get_fake_aligned_params(*_impl_params);
```

Comment:
How does it impact allocated buffer size if it's used in update_impl only? Don't we have illegal memory access?

Reply:
and nit: fake_params name may be confusing, maybe aligned_params or fake_aligned_params?

Reply:
>>How does it impact allocated buffer size if it's used in update_impl only? Don't we have illegal memory access?
Fake alignment is in updated_impl only. The required memory is applied in realloc_if_needed. 

>>and nit: fake_params name may be confusing, maybe aligned_params or fake_aligned_params?
I agree & updated.